.gitignore

```diff
@@ -32,3 +32,7 @@ Session.vim
 *~
 # auto-generated tag files
 tags
+
+### JetBrains files ###
+.idea/
+*.iml
```

Comment:
I think this is sth. that people should have in their global `.gitignore` really.